text/0000-discord.md

```diff
@@ -0,0 +1,170 @@
+**Explicit Content Filter**
+Since this is a public Discord server, we will be setting an explicit content filter- it will scan messages from all members without a role.
+
+**Categories and Channels**
+Community members will then have the option of visiting the "setup" channel and learning more about fine-grained controls, such as:
+
+- notifications
+- muting a channel
+- muting a category
+
+Because our goal is transparency, all channels will be visible. A lock icon will display if the user does not have the role necessary to join that channel.
+
+The following proposed initial category and channel list was chosen based on the current channel needs and evaluation of the channels with the most members on Slack. _Additional channels may be requested in the Admin/community-feedback channel._
+
+**Category/Channel List:**
+- (No Category)
+  - welcome (community guidelines are posted here)
+  - setup-profile (how to setup your profile)
+- Admin
+  - community-feedback (questions, comments, concerns, requests)
+  - security
+  - steering-committee 🔒 (locked to role “steering-committee”)
+  - news & announcements
+  - ember-jobs
+  - bots
+- Core Teams
+  - ember-js 🔒 (locked to role “core-js”)
+  - ember-data 🔒 (locked to role “core-data”)
+  - ember-cli 🔒 (locked to role “core-cli”)
+  - ember-learning 🔒 (locked to role “core-learning”)
+- Working on Ember
+  - ember-cli
+  - ember-data
+  - ember-engines
+  - ember-js
+  - glimmer-vm
+  - triage
+  - st-* (as needed)
+- Using Ember
+  - general-help
+  - learning-ember
+  - a11y
+  - backend
+  - internationalization
+  - jsonapi
+  - mobile
+  - ember-js
+  - ember-data
+  - ember-cli
+  - ember-engines
+  - fastboot
+  - ember-twiddle
+  - e-*
+- Supporting Ember
+  - documentation
+  - website
+  - marketing-and-advocacy
+  - infrastructure
+- Event-Chat
+  - EmberConf
+  - EmberCamp
+  - EmberFest
+  - Talks
+  - Other Conferences
+  - Meetup organizers
+- Social
+  - Water-cooler (random)
+  - Local-*
+  - Media (livestreams, videos, podcasts)
+  - Pets
+  - Women in Ember 🔒
+
+## How do we teach this?
+
+In addition to having a setup channel available upon login (with illustrated instructions), here are some links where community members can read more: 
+
+- [Discord Loves Open Source](https://discordapp.com/open-source)
+- [Discord Community Guidelines](https://discordapp.com/guidelines)
+- [How to use Discord](http://www.businessinsider.com/how-to-use-discord-the-messaging-app-for-gamers-2018-5)
+
+## Drawbacks
+
+### Supporting Learning vs Supporting Development
+
+There is some concern that there is already some confusion on Slack about where to get help learning/using Ember, and where to coordinate working on Ember. We need to have a clear delineation so that the folks who are spending their volunteer time to ship Ember features can continue to concentrate and do that.
+
+### Losing Community Members 
+
+There is some concern that we may lose some community members due to this move.
+
```

Comment:
I will add a section to address this :+1:

Reply:
> you can block the user.

That would require each user in the Ember discord to block the user?

Reply:
> A user would have to have already contacted you in a DM before their kick/ban to have the information to message you again

If you have the `@user#11234` for a single you can start a DM even after being removed from a group and without having DM'd in the past.

---

I know discord has a good track record of handling harassment, but just noting that the end moderation lies in Discord's hand vs Slack where that user no longer has access to DM individuals once removed from the group since users are tied to groups in slack.

Reply:
You can choose who can DM you, as well: 
![image](https://user-images.githubusercontent.com/4587451/42601835-9e4ad38e-852c-11e8-868b-0c0ff460ced1.png)